src/hotspot/cpu/x86/x86.ad

```diff
@@ -7500,13 +7501,24 @@ instruct rearrangeB(vec dst, vec shuffle) %{
   ins_pipe( pipe_slow );
 %}
 
-instruct rearrangeB_avx(vec dst, vec src, vec shuffle) %{
+instruct rearrangeB_avx(legVec dst, legVec src, vec shuffle, legVec vtmp1, legVec vtmp2, rRegP scratch) %{
   predicate(vector_element_basic_type(n) == T_BYTE &&
             vector_length(n) == 32 && !VM_Version::supports_avx512_vbmi());
```

Comment:
Predicate matches bail-out condition in match_rule_supported_vector(). Does it mean this code never used before?
So you are implementing it now. Right?

Reply:
Yes, this rule was not used before and I am implementing it now.